include/boost/url/impl/decoded_view.hpp

```diff
@@ -0,0 +1,229 @@
+//
+// Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_IMPL_DECODED_VIEW_HPP
+#define BOOST_URL_IMPL_DECODED_VIEW_HPP
+
+namespace boost {
+namespace urls {
+
+class decoded_view::iterator
+{
+    char const* begin_{nullptr};
```

Comment:
why do you use curly braces instead of equals? I think this is ok here but in general it can lead to trouble, especially with user-defined types.

Reply:
I don't know. I think I'm used to seeing curly braces more often but I don't care as much. Just changed it.